test/utils.ts

```diff
@@ -181,3 +182,61 @@ export function getCachedBatchId(): BatchId {
 
   return cachedBatchId
 }
+
+/**
+ * Returns FairOS API URL
+ */
+export function fairOSUrl(): string {
+  return process.env.FAIROS_API_URL || 'http://127.0.0.1:9090'
+}
+
+/**
+ * Waits until FairOS API return message about readiness
+ */
+export async function waitFairOS(): Promise<void> {
+  const url = fairOSUrl()
+  for (let i = 0; i <= 100; i++) {
+    let text
+    try {
+      text = (await axios.get(url)).data
+      // eslint-disable-next-line no-empty
+    } catch (e) {}
+
+    if (text === 'OK\n') {
+      return
+    } else if (text) {
```

Comment:
I get back 
```
FairOS-dfs
0.7.3-dev
/root/.fairOS/dfs
http://fdp-play-queen:1633
http://fdp-play-queen:1635
trace
:9090
:9091
localhost
[*]
```
from fairos for this request, that still does not seem wrong as a response, but this logic just tests against `OK\n`.
maybe you can check against response status instead?

Reply:
hm, on my side after `fdp-play start --pull --fresh --fairos` I receive `0.8.2-rc8-609cfb8`

But yes, checking the status is a more general approach for different versions